javascript-loops-practice/find-index.js

```diff
@@ -0,0 +1,11 @@
+/* exported findIndex */
+
+function findIndex(array, value) {
+  for (var i = 0; i < array.length; i++) {
+    if (array[i] === value) {
+      return i;
+    }
+  } if (value > array.length) {
```

Comment:
Why `value > array.length`? Is this function checking if the `value` given is larger than the length of the array to search?